dist/tools/openocd/openocd.sh

```diff
@@ -90,6 +90,8 @@
 # the target when starting a debug session. 'reset halt' can also be used
 # depending on the type of target.
 : ${OPENOCD_DBG_START_CMD:=-c 'halt'}
+# command used to reset the board
+: ${OPENOCD_CMD_RESET_RUN:=-c 'reset run'}
```

Comment:
```suggestion
: ${OPENOCD_CMD_RESET_RUN:="-c 'reset run'"}
```

Otherwise the single quotes get dropped and I get a `Unexpected command line argument: run`

Reply:
merged, thanks!